bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,106 @@
+
+- Consult the different parameters of your system, like temperature, current heating mode, water temperature, and many more.
+- Modify the functioning mode of your device: temperature set point, heating mode, and others.
+
+## Supported Things
+
+Support many different things as the thing type is handle by autodiscovery.
+
+Mainly, it will first discover the gateway.
+Currently test was done with the OZW672.x series.
+No test was conducted using the OZW772.x series, the code will currently not handle initialization of an OZW772 gateway.
+You can request support in the community forum, if you have the gateway model and want it to be supported.
+
+After, it will discover thing inside your PAC, mainly main controller of type RVS...
+Only test in real condition with RVS41.813/327 but should work with all other type as the access interface is standard.
+
+## Discovery
+
+Discovery of Gateway can be done using UPnP.
+Just switch off/on your gateway to make it annonce itself on the network.
+The gateway should appear in the Inbox a few minutes after.
+Be aware what you will have to modify the password in Gateway parameters just after the discovery to make it work properly.
+Be also aware that first initialization is a little long because the binding needs to read all the metadata from the device.
+
+Discovery of HVAC device have to be done through the Scan button inside the binding.
+Go to the Thing page, click on the "+" button, select the SiemensHVAC binding, and then click Scan.
+Your device should appear on the page after a few seconds.
+
+## Bridge Configuration
+
+Parameter       | Required       | Default        | Description
+----------------|----------------|----------------|------------------
+baseUrl         | yes            |                | The address of the OZW672 devices
+userName        | yes            | Administrator  | The user name to log into the OZW672
+userPass        | yes            |                | The user password to log into the OZW672
+
+## Channels
+
+Channels are auto-discovered, you will find them on the RVS things.
+They are organized the same way as the LCD screen of your PAC device, by top level menu functionality, and sub-functionalities.
+Each channel is strongly typed, so for example, for heating mode, openHAB will provide you with a list of choices supported by the device.
+
+Channel                | Description                                              | Type          | Unit     | Security Access Level   |  ReadOnly | Advanced
+-----------------------|----------------------------------------------------------|---------------|----------|-------------------------|-----------|-------------------
+control-boiler-approval  | Set Boiler Approval (`AUTO`, `OFF`, `ON`)                | String        |          |                         |  R/W      | true
+control-program         | Set Program (`OFF`, `NORMAL`, `WARMWATER`, `MANUAL`)     | String        |          |                         |  R/W      | true
```

Comment:
Can you reformat this table for proper alignment when viewing the source?

Reply:
Fix in next commit.

Reply:
It still look aligned from second colum.

Reply:
fix in next commit